diff --git a/toolchain/gcc/Config.version b/toolchain/gcc/Config.version
index e40d6b2..92033af 100644
--- a/toolchain/gcc/Config.version
+++ b/toolchain/gcc/Config.version
@@ -9,7 +9,7 @@ config GCC_VERSION_13
 config GCC_VERSION
 	string
 	default "11.3.0"	if GCC_VERSION_11
-	default "13.1.0"	if GCC_VERSION_13
+	default "13.3.0"	if GCC_VERSION_13
 	default "12.3.0"
 
 config GCC_USE_DEFAULT_VERSION
diff --git a/toolchain/gcc/common.mk b/toolchain/gcc/common.mk
index 31a1492..cdbf9fa 100644
--- a/toolchain/gcc/common.mk
+++ b/toolchain/gcc/common.mk
@@ -38,8 +38,8 @@ ifeq ($(PKG_VERSION),12.3.0)
   PKG_HASH:=949a5d4f99e786421a93b532b22ffab5578de7321369975b91aec97adfda8c3b
 endif
 
-ifeq ($(PKG_VERSION),13.1.0)
-  PKG_HASH:=61d684f0aa5e76ac6585ad8898a2427aade8979ed5e7f85492286c4dfc13ee86
+ifeq ($(PKG_VERSION),13.3.0)
+  PKG_HASH:=0845e9621c9543a13f484e94584a49ffc0129970e9914624235fc1d061a0c083
 endif
 
 PATCH_DIR=../patches-$(GCC_MAJOR_VERSION).x
 
